dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.2.6.2/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java

```diff
@@ -502,7 +502,7 @@ private void cacheInput(Message outMessage) {
                 }
             }
             if (in != null) {
-                in.cacheInput();
+                //in.cacheInput();
```

Comment:
If you are considering changing this in the 2.6 code, you will want to run the JAXWS 2.2 tests as well.